packages/vm/src/wasm_backend/store.rs

```diff
@@ -55,10 +55,15 @@ pub fn make_compile_time_store(memory_limit: Option<Size>) -> Store {
 }
 
 /// Created a store with no compiler and the given memory limit (in bytes)
-/// If memory_limit is None, no limit is applied.
+/// If memory_limit is 0, no limit is applied.
 pub fn make_runtime_store(memory_limit: Size) -> Store {
```

Comment:
The docs were wrong before as the limit was not optional. As you can see in many other places, we use `Option<Size>` for optional memory limits. Here we don't have the option because I thought we always need a limit for the runtime store.

Making it optional is fine for the new use case, but let's use `memory_limit: Option<Size>`